perfkitbenchmarker/scripts/database_scripts/file_to_plot.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
```

Comment:
Good point. It was a little too literal. Changed filename and all references to plot_sysbench_results.py